ci/release/update-version.sh

```diff
@@ -51,3 +52,8 @@ sed_runner 's/'"CUDA_DATAFRAME VERSION .* LANGUAGES"'/'"CUDA_DATAFRAME VERSION $
 # RTD update
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/cudf/source/conf.py
 sed_runner 's/version = .*/version = '"'${NEXT_SHORT_TAG}'"'/g' docs/nvstrings/source/conf.py
+
+# bump rmm
+for FILE in conda/environments/*.yml; do
+  sed_runner "s/rmm=${CURRENT_SHORT_TAG}/rmm=${NEXT_SHORT_TAG}/g" ${FILE};
+done
```

Comment:
Linebreak at end of file :)
```suggestion
done

```